mender/templates/gui-deploy.yaml

```diff
@@ -62,7 +62,7 @@ spec:
 
         env:
         - name: INTEGRATION_VERSION
-          value: "3.5.0"
+          value: "3.5.1"
         - name: MENDER_VERSION
           value: "3.5.0"
```

Comment:
@mzedel: Can you confirm that this is correct? The Mender client version is still 3.5.0, IOW the Debian package. But the QEMU image is still part of the release, and is therefore tagged as `mender-3.5.1`. Will it treat it correctly?

Reply:
Looks correct to me!
The client version should only be relevant for the version information in the lower left corner, the device connection snippets all rely on what the install script installs (afaik the latest stable release) or use the untagged docker image (so `latest`)...

Reply:
Ok, that should all work! Thanks!